src/libopensc/card-openpgp.c

```diff
@@ -140,6 +130,7 @@ static int		pgp_get_pubkey(sc_card_t *, unsigned int, u8 *, size_t);
 static int		pgp_get_pubkey_pem(sc_card_t *, unsigned int, u8 *, size_t);
 
 
```

Comment:
clang-format suggested the following change:


```
-
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-openpgp.c -->